### PR TITLE
William/view

### PR DESCRIPTION
**Pull request by willclaros**

### CODE REVIEW

File: FileSearch/src/com/fundation/search/view/MainPanel.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ */
+package com.fundation.search.view;
+
+import javax.swing.BoxLayout;
+import javax.swing.JPanel;
+
+/**
+ * Class that creates intermediate panels.
+ *
+ * @author William Claros Revollo
```
Comment (erikAlberto) [inline FileSearch/src/com/fundation/search/view/MainPanel.java]: add after name - AT-[07].
and add  @version 1.0.

File: FileSearch/src/com/fundation/search/view/SearchFrame.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     * Constructor to create the main Frame.
+     */
+    public SearchFrame() {
+        super("File Search");
+        settingMainFrame();
+        initComponent();
+
+    }
+
```
Comment (valetodo) [inline FileSearch/src/com/fundation/search/view/SearchFrame.java:39]: Missing comments Add

File: FileSearch/src/com/fundation/search/view/SearchFrame.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    private void settingMainFrame(){
+        setExtendedState(JFrame.MAXIMIZED_BOTH);
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+        setLayout(null);
+        setVisible(true);
+    }
+
+    private void initComponent(){
```
Comment (valetodo) [inline FileSearch/src/com/fundation/search/view/SearchFrame.java:47]: Same as above

File: FileSearch/src/com/fundation/search/view/SearchFrame.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        panelGeneral = new MainPanel(1400, 700);
+        this.setContentPane(panelGeneral);
+    }
+
+    /**
+     * Main method to initialize the View.
```
Comment (valetodo) [inline FileSearch/src/com/fundation/search/view/SearchFrame.java:53]: incomplete descriptions

**Review by valetodo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**